apps/api/src/app/inbox/inbox.controller.ts

```diff
@@ -313,10 +316,26 @@ export class InboxController {
   }
 
   @UseGuards(AuthGuard('subscriberJwt'))
-  @Patch('/preferences/:workflowId')
+  @Patch('/preferences/bulk')
```

Comment:
Any specific reason for PATCH vs POST? because of the partial preference object?

Reply:
yes it's partial update and to align with the existing single update